src/services/PrivateRoute.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { useAuth } from 'services/AuthProvider';
+
+// eslint-disable-next-line react/prop-types
+export default function PrivateRoute({ component: Component, ...rest }) {
+  const { currentUser } = useAuth();
+
+  return (
+    <Route
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...rest}
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      render={(props) => (currentUser ? <Component {...props} /> : <Redirect to="/login" />)}
+    />
+  );
+}
```

Comment:
Private route is a `component` and not a service. Let's transfer this into the `components` directory.

Also change the path of the redirect to use constant routes and not strings